### DISCUSSION TITLE
Lazer - Standard - gameplay - objects - Is it to late to add "held circle" to lazer?

### DISCUSSION THREAD

**Discussion by MilesNaismith1** [ppy/osu#16126] [Ideas] [answered]
There was previously discussion about adding "held circles" to the game. I am not sure when and where it was taking place, i know that it did not result in the thing being added. Would it be possible to revisit the idea and consider adding  it to lazer still? From the perspective of a player, this is already used in mapping in a form of slider that is very very slow so it looks like a circle. Wouldn't it make sense to actually allow mappers to use the circles with the property of a slider where u need to tap and hold a key for a certain amount of time? For me personally as a player it would make the case of held notes more readable in gameplay, as well as introduce new element/mechanic for the gameplay of standard which could potentially lead to some cool beatmaps being created.

**Comment by smoogipoo** [comment, marked as answer]
Yep, it's on the list of things we want to make happen.
